### PR TITLE
Refactoring contributors section

### PR DESCRIPTION
**Pull request by AndreiBarari**
DO NOT APPROVE ... yet!

### CODE REVIEW

File: style/scss/_team.scss [hunk 1]
```diff
@@ -18,12 +18,13 @@ $tablet-font-size-h4: 2rem;
...
 
 .team {
+  padding: 3rem 1rem;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  gap: 3rem;
   background-color: #f3f3f3;
-  padding: 0 3.2rem 2.4rem;
```
Comment (talidag) [inline style/scss/_team.scss:26]: The padding top should be consistent across all sections (7rem)

File: style/scss/_team.scss [hunk 2]
```diff
@@ -33,35 +34,101 @@ $tablet-line-height-p: 2rem;
...
+      align-items: center;
+      justify-content: center;
+      flex-wrap: wrap;
+      gap: 0.75rem;
+      z-index: 10;
+
+      .team-nav-btn {
+        font-size: 1.6rem;
+        font-weight: 600;
+        letter-spacing: 1px;
```
Comment (talidag) [inline style/scss/_team.scss:75]: Please remove this letter spacing, it should be consistent across all buttons (0px)

File: style/scss/_team.scss [hunk 3]
```diff
@@ -33,35 +34,101 @@ $tablet-line-height-p: 2rem;
...
+        padding: 0.5rem 0.75rem;
```
Comment (talidag) [inline style/scss/_team.scss:76]: I think it looks better with 0.5rem 1rem padding but feel free to check it out.

File: teamMembers.json [hunk 4]
```diff
@@ -0,0 +1,338 @@
+[
+  {
+    "name": "Adriana Cionca",
+    "avatar": "./assets/contributors/adriana-cionca.webp",
```
Comment (talidag) [inline teamMembers.json:4]: I feel like for some people the avatar is way to high quality and it looks weird. For example for almost all members in FRONTEND Team. Is there something you can do about this?

File: teamMembers.json [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+      "github": "https://github.com/AdrianaCionca",
+      "discord": "https://discord.com/users/1064933385878904954"
+    }
+  },
+  {
+    "name": "Talida Ganciu",
+    "avatar": "./assets/contributors/talida-ganciu.webp",
+    "position": "Front-end Lead",
+    "team": "Front-end",
+    "teamLead": true,
```
Comment (talidag) [inline teamMembers.json:19]: Maybe we can do a bit more to differentiate the lead from the team.

File: style/scss/_team.scss (hunk 1 above)
Comment (talidag) [inline style/scss/_team.scss:26]: Please also add a bit of margin top. I feel like the section above doesnt have a padding-bottom.

File: style/scss/_team.scss [hunk 6]
```diff
@@ -33,35 +34,101 @@ $tablet-line-height-p: 2rem;
...
     width: 23rem;
   }
 
-  p {
+  &__description {
     width: auto;
     font-size: 1.6rem;
     line-height: 130%;
     text-align: center;
+    text-wrap: balance; // Wrap text evenly
```
Comment (talidag) [inline style/scss/_team.scss:50]: Looks very nice now! Good job

File: teamMembers.json [hunk 7]
```diff
@@ -0,0 +1,338 @@
+[
```
Comment (talidag) [inline teamMembers.json:1]: AWESOME JOB WITH THE JSON!

File: style/scss/_team.scss (hunk 1 above)
Comment (AndreiBarari) [reply style/scss/_team.scss:26]: Added 7 rem padding top to keep consistency over all sections

File: style/scss/_team.scss (hunk 2 above)
Comment (AndreiBarari) [reply style/scss/_team.scss]: Removed letter spacing to be consisntent with all the buttons on the website

File: style/scss/_team.scss (hunk 3 above)
Comment (AndreiBarari) [reply style/scss/_team.scss]: Updated the padding, it works better with .5 and 1 rem padding

File: teamMembers.json (hunk 4 above)
Comment (AndreiBarari) [reply teamMembers.json:4]: I will look into it

**Review by talidag — APPROVED** [review]

**Pull request merged**